contracts/governance/AxelarMultisigMintLimiter.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { SafeNativeTransfer } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/SafeTransfer.sol';
+import { IMultisigExecute } from '../interfaces/IMultisigExecute.sol';
+import { MultisigBase } from '../auth/MultisigBase.sol';
+
+contract AxelarMultisigMintLimiter is MultisigBase, IMultisigExecute {
+    using SafeNativeTransfer for address;
+
+    constructor(address[] memory accounts, uint256 threshold) {
+        _rotateSigners(accounts, threshold);
+    }
+
+    function execute(
```

Comment:
Worth adding a separate `MultisigExecute` contract that contains this for reusability (we'll have a need to deploy a standard multisig contract)

Reply:
I'm thinking we can just rename this contract to `Multisig` and it can be used generically for mint limits, etc.